modules/coreos-layering-configuring-on.adoc

```diff
@@ -6,24 +6,21 @@
 [id="coreos-layering-configuring-on_{context}"]
 = Using on-cluster layering to apply a custom layered image
 
-To apply a custom layered image to your cluster by using the on-cluster build process, make a `MachineOSConfig` custom resource that includes a Containerfile, a machine config pool reference, repository push and pull secrets, and other parameters as described in the prerequisites.
+To apply a custom layered image to your cluster by using the on-cluster build process, make a `MachineOSConfig` custom resource that specifies the Containerfile to build, the machine config pool to associate the build, where the final image should be pushed and pulled from, the push and pull secrets to use, and other parameters as described in the prerequisites.
 
-When you create the object, the Machine Config Operator (MCO) creates a `MachineOSBuild` object and a `machine-os-builder` pod. The build process also creates transient objects, such as config maps, which are cleaned up after the build is complete. 
+When you create the object, the Machine Config Operator (MCO) creates a `MachineOSBuild` object and a `machine-os-builder` pod. The build process also creates transient objects, such as config maps, which are cleaned up after the build is complete.
 
 When the build is complete, the MCO pushes the new custom layered image to your repository for use when deploying new nodes. You can see the digested image pull spec for the new custom layered image in the `MachineOSBuild` object and `machine-os-builder` pod.
 
 You should not need to interact with these new objects or the `machine-os-builder` pod. However, you can use all of these resources for troubleshooting, if necessary.
 
 You need a separate `MachineOSConfig` CR for each machine config pool where you want to use a custom layered image.
 
-:FeatureName: On-cluster image layering
-include::snippets/technology-preview.adoc[]
-
 .Prerequisites
 
-* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see "Enabling features using feature gates".
+* You have a copy of the global pull secret in the `openshift-machine-config-operator` namespace that the MCO needs to pull the base operating system image.
```

Comment:
We can remove this line because the MCO will automatically clone the global pull secret into the MCO namespace and keep it up-to-date.

Reply:
I am going to leave this line in this PR, as I want to apply the changes to all OCL versions. I am address these issues in separate PRs.